test/stat_tracker_test.rb

```diff
@@ -4,6 +4,18 @@
 
 class StatTrackerTest < MiniTest::Test
 
+  @game_path = './data/games.csv'
```

Comment:
The way this is written works and it cuts down the test time, by A LOT. But, is this the proper way to be using this? Should @@stat_tracker be a class variable? Do @games, @teams,a nd @game_teams need to be instance variables? They aren'y called upon anywhere else within the StatTrackerTest class.